### PR TITLE
Add my card name

### PR DESCRIPTION
**Pull request by Christiano112**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:118]: Please change this comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->`
to `<!-- ________ Your_Name card START ________ -->`

File: index.html [hunk 2]
```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#https://www.educative.io/explore/early-access" target="_blank" title="Educative">
+                  Educative
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:147]: Please change this comment as well.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @Christiano112. Thank you for contributing to this project.
You should make a couple of small changes to your code before we can merge your pull request. 
These comments are very important for everyone when they read the HTML file, so they would be able to identify your code. 
Once you've made these changes, we will be able to review and merge your pull request in a flash.

File: index.html (hunk 1 above)
Comment (Christiano112) [reply index.html]: Done.

**Comment by Christiano112** [comment]
I have made the require changes

**Review by dilshankarunarathne — APPROVED** [review]

**Comment by dilshankarunarathne** [comment]
Congratulations 🎉❤ @Christiano112.
Your pull request was merged. 
You will be able to see your card on the [project website](https://syknapse.github.io/Contribute-To-This-Project/).

**Pull request merged**